README.md

```diff
@@ -16,7 +16,7 @@ Options:
   --device PATH_OR_URL           [required]
   --baudrate INTEGER             [default: 115200]
   --bootloader-baudrate INTEGER  [default: 115200]
-  --cpc-baudrate INTEGER         [default: 115200]
+  --cpc-baudrate INTEGER         [default: 115200, 460800]
```

Comment:
Can we switch to `460800, 115200` directly? This way, once we use this version of `universal-silabs-flasher` it would only slow down the first startup and subsequent would use the new baud rate.